responsive-test-utils/src/test/java/dev/responsive/kafka/api/TopologyTestDriverExampleTest.java

```diff
@@ -45,7 +45,7 @@ public void shouldRunWithoutResponsiveConnection() {
     props.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.StringSerde.class);
     props.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.StringSerde.class);
 
-    final Topology topo = topology(new TestStoreDriver());
+    final Topology topo = topology();
```

Comment:
I realize this test doesn't really do anything anymore, we'll need to rethink the TestStoreDriver approach now that we have static store factories. We can put this on the list of things to discuss

Reply:
For reference, here's a ticket to address this: https://www.notion.so/responsivedev/Write-a-test-driver-client-with-in-memory-backend-d2e674cf264147d0b8fa83934260fda7?pvs=4